app/exporter/celery_events_exporter.py

```
@@ -0,0 +1,168 @@
+            log.error("Celery event stream error", exc_info=e, extra={"error": str(e)})
+        finally:
+            if connection:
+                connection.close()
+            log.info("Event consumer loop exited")
+
+    def _handle_event(self, event: dict) -> None:
+        """Handle incoming Celery event (called by event receiver)."""
+        if self.running:
+            try:
+                self.event_queue.put_nowait(event)
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)